hazelcast-integration/spring-hibernate-2ndlevel-cache/src/main/resources/applicationContext-hazelcast.xml

```diff
@@ -17,9 +17,10 @@
 
 	        <hz:network port="${hz.network.port}" port-auto-increment="${hz.network.port.auto.increment}">
 	            <hz:join>
-	                <hz:tcp-ip enabled="${hz.tcp.ip.enabled}">
-	                    <hz:members>${hz.members}</hz:members>
-	                </hz:tcp-ip>
+			<hz:multicast enabled="false" multicast-group="224.2.2.3" multicast-port="54327"/>
```

Comment:
you must change this file: https://github.com/hazelcast/hazelcast-code-samples/blob/master/hazelcast-integration/spring-hibernate-2ndlevel-cache/src/main/resources/hazelcast-config.properties
and these changes should be belong to another pr. (with hibernate 2nd level cache small fix)